Add small_formatting argument to file_size

zwimer opened the pull request:
Fixes: https://github.com/staticdev/human-readable/issues/691 by adding in a `small_formatting` argument which handles formatting when `value` is small enough that its suffix would be `B`, `Byte`, or `Bytes`.

staticdev commented:
@zwimer thanks for your contribution. Could you please add the necessary tests so i can approve it?

staticdev commented on src/human_readable/files.py:
```diff
@@ -30,18 +34,19 @@ def file_size(
         suffixes = (" kB", " MB", " GB", " TB", " PB", " EB", " ZB", " YB")
 
     base = 1024 if (gnu or binary) else 1000
+    fmt = small_formatting if value < base else formatting
```
This linr is not covered by tests anymore.

zwimer commented:
I added test cases and noticed that among all the suffixes, there was one `kB` where everything else was capitalized it so I fixed that by changing it to `KB`.

zwimer commented:
@staticdev Is there anything else I can do to help get this merged?

staticdev commented:
@zwimer merged, thanks.
